ci/Debian11Container.Jenkinsfile

```diff
@@ -0,0 +1,303 @@
+pipeline {
+    environment {
+    	// on m1 macs, this is a symlink that must be updated. see wiki.
+        // VERSION = sh(returnStdout: true, script: "${env.NODE_PATH}/node -p -e \"require('./package.json').version\" | tr -d \"\n\"")
+		VERSION = "9001.99.1"
```

Comment:
I suspect this should be changed before merging

Reply:
I believe it is in the second commit, b928bd3c46ad93827d90caa4f2b6ce9b23ad5025.